renv.lock

```diff
@@ -65,25 +65,6 @@
       "Repository": "CRAN",
       "Hash": "f3ca785924863b0e4c8cb23b6a5c75a1"
     },
-    "admiral.test": {
-      "Package": "admiral.test",
-      "Version": "0.2.0",
-      "Source": "GitHub",
-      "RemoteType": "github",
-      "RemoteHost": "api.github.com",
-      "RemoteUsername": "pharmaverse",
-      "RemoteRepo": "admiral.test",
-      "RemoteRef": "devel",
-      "RemoteSha": "b1f4a44a76410e5ef285f607e038f4c5b9f02dc9",
-      "Hash": "c6002d9280db2b0c00c49320fe00684b"
-    },
-    "admiraldev": {
```

Comment:
I think by removing this, code containing assert_xxx functions such as 120-122, 361-364, and 416 in `R/period_dataset.R` won't run properly therefore function won't work?

Reply:
admiral packages like admiraldev or admiral.test should not be in `renv.lock` because we need different versions depending on where the code is running. For development and merging feature branches into `devel` we use the devel version of admiraldev and admiral.test. For merging into `pre-release` or `main` we use the main version of admiraldev and admiral.test. The CI/CD workflows in github use staged dependencies to handle this.
Locally you need to call
```
renv::restore()
remotes::install_github("pharmaverse/admiral.test", ref = "devel")
remotes::install_github("pharmaverse/admiraldev", ref = "devel")
```
to set up your working environment.

@bms63 , @thomas-neitmann , should we create a function or addin which set up the environment for developing? This could also include an option to set up the environment as used by the R-CMD checks. What do you think?

Reply:
There is already an RStudio addin from the {staged.dependencies} package which install the packages specified in the `staged_dependencies.yaml` file.

Reply:
@thomas-neitmann , yes, we could use this. Unfortunately, it is not very user friendly. When I tried it the job took more than 10 minutes and the messages are confusing. After some seconds the message was
```
Copying local dir /home/bceuser/bundfuss/admiral to cache dir /home/bceuser/bundfuss/.staged.dependencies/packages_cache/local_pharmaverse_admiral_5c04f375bd8b875e0d1733a6f58ba11c
```
After that nothing happened for about 10 minutes. Then the installation of the upstream dependencies started and failed with the same error we have also observed in some of the workflows (admiraldev can not be loaded because assertthat is not available).